### ISSUE TITLE
Configuration field lacks translation

### ISSUE THREAD

**Issue by petrenkonikita112263** [xwikisas/application-task#71]
After installing latest version of Task Manager (Pro), navigate into Admininstartion section of it (XWiki Preferences → Other → Task Manager). You'll see NOTSKIPPEDEVENTS with package name of contrib extensions.

![image](https://github.com/xwikisas/application-task/assets/56109799/f2fc4373-f598-4b3b-92f5-2fe9bc4736ef)

It's not specified in [documentation](https://store.xwiki.com/xwiki/bin/view/Extension/TaskApplication#documentation) for what this setting is responsible for. Moreover, Admin can edit this field due to the available CKEditor menu toolbar and even insert other macros.

**Comment by petrenkonikita112263** [comment]
Introduce in #59

**Comment by trrenty** [comment]
It's not a bug. It's a feature :P

Indeed, the setting lacks documentation. Thanks to your notice, I've updated it https://store.xwiki.com/xwiki/bin/view/Extension/TaskApplication#HSkiptaskmacroevaluationduringFoldEvents

Also, thanks to your issue, I noticed that the label does not have a translation. So I'll rename the issue accordingly and made the necessary modification.

> Moreover, Admin can edit this field due to the available CKEditor menu toolbar and even insert other macros.

The property is of type **textarea** which will be displayed using the CKEditor. I don't think it is an issue.
